src/clj/game/core/damage.clj

```diff
@@ -122,29 +123,43 @@
       (- (or (when-not unpreventable (get-in @state [:damage :damage-prevent dtype])) 0))
       (max 0)))
 
+(defn check-damage-prevention
+  "for a preventable damage instance, handles all damage prevention effects that a player can use for it"
+  ([state side eid type n args player]
+   (let [interrupts (get-prevent-list state player type)
+         other-player (if (= player :corp) :runner :corp)
+         already-prevented (or (get-in @state [:damage :damage-prevent type]) 0)]
+     (if (and (cards-can-prevent? state player interrupts type)
```

Comment:
```suggestion
     (if (and (cards-can-prevent? state player interrupts type nil {:side side})
```

This should let you say `:req (req (= :corp (:side target)))` in Prana Condensor